faq/tidb.md: update the wording about transaction size (#1876)
pingcap/docs: faq/tidb.md

#### The error message `transaction too large` is displayed

Due to the limitation of the underlying storage engine, each key-value entry (one row) in TiDB should be no more than 6MB.

> Distributed transactions need to conduct two-phase commit and the underlying implementation performs Raft replication. If a transaction is very large, the commit process would be quite slow and this transaction is more likely to conflict with other transactions. Moreover, the rollback of a failed transaction leads to an unnecessary performance penalty. To avoid these problems, we limit the default transaction total size of key-value entries to no more than 100MB. If you need larger transactions, change `txn-total-size-limit` in the TiDB configuration file. The maximum value of this configuration item is up to `10G`. The actual limitation might also depend on the hardware of the RAM.

Reviewer: ```suggestion
Distributed transactions need two-phase commit and the underlying implementation performs the Raft replication. If a transaction is very large, the commit process would be quite slow and this transaction is more likely to conflict with other transactions. Moreover, the rollback of a failed transaction leads to an unnecessary performance penalty. To avoid these problems, we limit the total size of key-value entries to no more than 100MB in a transaction by default. If you need larger transactions, change `txn-total-size-limit` in the TiDB configuration file. The maximum value of this configuration item is up to `10G`. The actual limitation might also depend on the hardware of the RAM.
```

#### The error message `transaction too large` is displayed

> As distributed transactions need to conduct two-phase commit and the underlying implementation performs Raft replication. If a transaction is very large, the commit process would be quite slow and it's more likely to conflict with other transactions. Morever, the rollback of transactions lead to unnecessary performance penalty. To avoid this problem, we limit the transaction size:

Reviewer: ```suggestion
Distributed transactions need to conduct two-phase commit and the underlying implementation performs Raft replication. If a transaction is very large, the commit process would be quite slow and this transaction is more likely to conflict with other transactions. Moreover, the rollback of a failed transaction leads to an unnecessary performance penalty. To avoid these problems, we limit the transaction size:
```

- A transaction is limited to 5000 SQL statements (by default)
- Each Key-Value entry is no more than 6MB

There are [similar limits](https://cloud.google.com/spanner/docs/limits) on Google Cloud Spanner.

> By default we limit the total size of Key-Value entries to no more than 100MB, if you need larger transactions, you can change `txn-total-size-limit` in the TiDB configuration file, the maximum value could be up to 10G, do not forget the actual limitation may also depend on the hardware of the RAM.

Reviewer: ```suggestion
By default, we limit the total size of key-value entries to no more than 100MB. If you need larger transactions, change `txn-total-size-limit` in the TiDB configuration file. The maximum value of this configuration item is up to `10G`. The actual limitation might also depend on the hardware of the RAM.
```
